html/changelog.html

```diff
@@ -25,7 +25,8 @@
 		<td valign='top'>
 			<div align='center'><font size='3'><b>Paradise Station</b></font></div>
 			
-			<p><div align='center'><font size='3'><a href="https://www.paradisestation.org/forum/">Forum</a> | <a href="https://www.paradisestation.org/wiki/index.php/Main_Page">Wiki</a> | <a href="https://github.com/ParadiseSS13/Paradise">Source</a></font></div></p>
+			<p><div align='center'><font size='3'><a href="https://www.paradisestation.org/forum">Forum</a> | <a href="https://www.paradisestation.org/wiki/index.php/Main_Page">Wiki</a> | <a href="https://github.com/ParadiseSS13/Paradise">Source</a></font></div></p>
+			<font size='2'><b>Visit our Discord channel:</b><a href='https://discordapp.com/invite/nuqD478<'>-Click Here-</a></font>

```

Comment:
This was added in because the person who removed it before never removed it from
`./html/templates/header.html`. I can either remove it from both or leave it like this.